### DISCUSSION TITLE
3 phases with Webasto Next?

### DISCUSSION THREAD

**Discussion by puskasl1** [lbbrhzn/ocpp#1158] [Q&A]
All Controls work fine with this charger, however, no data is shown for those measurands that are received with the "phase" attribute, like "Voltage" or "Current.Import". In the log, I can see the values, but in HA there is no attribute available. Has someone managed to solve a similar issue?

**Comment by stijnilsen** [comment]
Hi! I'm trying to connect my Webasto Next. But initially I could not get the charger to be shown as a device. Suddenly it showed up, and I got the measurands to read out something. Like the firmware version. But the values were unavailable for hours, then OK for a few minutes, ...
Definitely not correct.
I have configure the webasto OCPP address to: ws://192.168.2.86:9000, where 192.168.2.86 is the IP address of the raspberry Pi where I run HA. Do you have any suggestions on:
* How you configured the Webasto Next?
* How you managed to get it show up as a device in the OCPP integration?
@puskasl1 , a big thank you for quickly helping me
